Fix typos, and added sentence about json result

dorely103 opened the pull request:
General edits such as format, missing words, and parts that needed clarifying

imanehaf reviewed (changes requested):
Nice additions here! 
Again minor comments:
- What is raphtory_example in Line28?? [Same comment about pointing to the right folder]
- I would keep `hops` in Line67 since it's the graphy term that's usually used.
- I'm not sure what you wanted to say in Line117. ` with the total size of the network divided by the number of characters that directly interacted with Gandalf (in this case, 24 divided by 9)` The only data output is the size of the returned network (24) and direct connections (9) ...
- Code for the last query is missing `rg.viewQuery(SixDegreesOfGandalf(),timestamp = 5000,arguments)`

dorely103 commented:
Thanks Imane! 
- The raphtory_example is the cloned directory from the installation guide but I've rewritten it so now it is more clear
- Changed back to hops
- I've rewritten this part to explain that there are total 24 nodes and 9 are directly connected to gandalf
- Added the missing code under the range query
